tests/framework/Models/External/tests

```diff
@@ -9,6 +9,11 @@
   input = 'serialize_ext_model_and_use.xml'
   csv = 'SerializeWrkd/samples_out.csv'
  [../]
+ [./pyomo_serialization]
+  type = 'RavenFramework'
+  input = 'serialize_pyomo.xml'
+  output = 'SerializePyomo/rom_out.py'
```

Comment:
I think we need to test the generated file, otherwise we will not know if it is broken or not in future development

Reply:
In order to test this file, pyomo and Pynumero are required installs. Then, the rom_out.py and the associated pickled file can be used to run pyomo. Are you suggesting to add a test which requires pyomo to be part of the raven libraries?

Reply:
I think we can add pyomo as an optional library for RAVEN. Let's have a discussion on Monday meeting